app/controllers/workflows_controller.rb

```diff
@@ -59,6 +57,8 @@ def update
 
   # add a workflow to an object if the workflow is not present in the active table
   def create
+    cocina = maybe_load_cocina(params[:item_id])
```

Comment:
Will this always be an item? Or can it be an APO or collection? I ask mostly to help guide what variable name we use here.

Reply:
It could be a Collection or a DRO, both have workflows.

Reply:
OK. Should we call it `cocina_object` then?